src/main/client/bullet/default-bullet.ts

```diff
@@ -61,25 +63,30 @@ export class Bullets extends Phaser.Physics.Arcade.Group {
       });
 
       this.bulletSounds = [
-         scene.sound.add(Keys.BASIC_GUN_SOUND, {
+         ...this.varySound(scene, Keys.GUN_SOUND_1),
+         ...this.varySound(scene, Keys.GUN_SOUND_2),
+         ...this.varySound(scene, Keys.GUN_SOUND_3),
+         ...this.varySound(scene, Keys.GUN_SOUND_4),
+      ];
+   }
+
+   private varySound(scene: Scene, key: string): BaseSound[] {
+      return [
+         scene.sound.add(key, {
             volume: 0.3,
          }),
-         scene.sound.add(Keys.BASIC_GUN_SOUND, {
+         scene.sound.add(key, {
             volume: 0.3,
             detune: -100,
          }),
-         scene.sound.add(Keys.BASIC_GUN_SOUND, {
+         scene.sound.add(key, {
             volume: 0.3,
             detune: -50,
          }),
-         scene.sound.add(Keys.BASIC_GUN_SOUND, {
+         scene.sound.add(key, {
             volume: 0.3,
             detune: 50,
          }),
-         scene.sound.add(Keys.BASIC_GUN_SOUND, {
-            volume: 0.3,
```

Comment:
why is 100 detune removed

Reply:
There was no 100 detune. I kept the values